tests/tests_sbd_all_options.yml

```diff
@@ -31,6 +31,10 @@
         - name: Set SBD devices and watchdogs
           set_fact:
             ha_cluster:
+              sbd_watchdog_modules:
+                - softdog
```

Comment:
The problem here is that `softdog` module is loaded by `test_setup_sbd.yml` above. So you don't actually test that the role loaded it. However, since `softdog` is the only watchdog available in CI, we need to stick to it.

How about adding
```
vars:
  __test_disable_modprobe: true
```
to `Set up test environment for SBD` task and 
```
when:
  - not (__test_disable_modprobe | d(false))
```
to `Load softdog module for SBD to have at least one watchdog` task in `test_setup_sbd.yml`?